src/d3a/models/base.py

```diff
@@ -25,12 +25,10 @@
 
 
 class AreaBehaviorBase:
-    """Base class used by area behaviour defining classes (`BaseStrategy`, `BaseAppliance`)"""
+    """Base class used by area behaviour defining classes `BaseStrategy`"""
     def __init__(self):
-        # `area` is the area we trade in
```

Comment:
I think that these comments are useful, please leave these here. Thanks!

Reply:
Agreed! I thought they are just wrong, but no, they are right!. Will re-add them